packages/shopify-app-remix/src/__tests__/test-helper.ts

```diff
@@ -33,203 +32,3 @@ export function testConfig(
-        email: 'test@test.test',
-        email_verified: true,
-        first_name: 'Test',
-        last_name: 'User',
-        locale: 'en-US',
-      },
-    };
-  }
-
-  const session = new Session({
-    id,
-    shop: TEST_SHOP,
-    isOnline,
-    state: 'test',
-    accessToken: 'totally_real_token',
-    scope: 'testScope',
-    ...overrides,
-  });
-  await sessionStorage.storeSession(session);
-
-  return session;
-}
-
-export function signRequestCookie({
-  request,
-  cookieName,
-  cookieValue,
-}: {
-  request: Request;
-  cookieName: string;
-  cookieValue: string;
-}) {
-  const signedCookieValue = createTestHmac(cookieValue);
-
-  request.headers.set(
-    'Cookie',
-    [
-      `${cookieName}=${cookieValue}`,
-      `${cookieName}.sig=${signedCookieValue}`,
-    ].join(';'),
-  );
-}
-
-export function expectBeginAuthRedirect(
-  config: ReturnType<typeof testConfig>,
-  response: Response,
-) {
-  expect(response.status).toEqual(302);
-
-  const {hostname, pathname, searchParams} = new URL(
-    response.headers.get('location')!,
-  );
-
-  expect(hostname).toBe(TEST_SHOP);
-  expect(pathname).toBe('/admin/oauth/authorize');
-  expect(searchParams.get('client_id')).toBe(config.apiKey);
-  expect(searchParams.get('scope')).toBe(config.scopes!.toString());
-  expect(searchParams.get('redirect_uri')).toBe(
-    `${config.appUrl}/auth/callback`,
-  );
-  expect(searchParams.get('state')).toStrictEqual(expect.any(String));
-}
-
-interface ExpectExitIframeRedirectOptions {
-  shop?: string;
-  host?: string | null;
-  destination?: string;
-}
-
-export function expectDocumentRequestHeaders(
-  response: Response,
-  isEmbeddedApp = true,
-) {
-  const headers = response.headers;
-
-  if (isEmbeddedApp) {
-    expect(headers.get('Content-Security-Policy')).toEqual(
-      `frame-ancestors https://${encodeURIComponent(
-        TEST_SHOP,
-      )} https://admin.shopify.com https://*.spin.dev;`,
-    );
-    expect(headers.get('Link')).toEqual(
-      `<${APP_BRIDGE_URL}>; rel="preload"; as="script";`,
-    );
-  } else {
-    expect(headers.get('Content-Security-Policy')).toEqual(
-      `frame-ancestors 'none';`,
-    );
-  }
-}
-
-export function expectExitIframeRedirect(
-  response: Response,
-  {
-    shop = TEST_SHOP,
-    host = BASE64_HOST,
-    destination = `/auth?shop=${shop}`,
-  }: ExpectExitIframeRedirectOptions = {},
-) {
-  expect(response.status).toBe(302);
-
-  const {pathname, searchParams} = new URL(
-    response.headers.get('location')!,
-    APP_URL,
-  );
-  expect(pathname).toBe('/auth/exit-iframe');
-
-  expect(searchParams.get('shop')).toBe(shop);
-  expect(searchParams.get('exitIframe')).toBe(destination);
-
-  if (host) {
-    expect(searchParams.get('host')).toBe(host);
-  }
-}
-
-export function expectLoginRedirect(response: Response) {
-  expect(response.status).toBe(302);
-
-  const {pathname} = new URL(response.headers.get('location')!, APP_URL);
-  expect(pathname).toBe('/auth/login');
-}
```

Comment:
Is the thinking here that `/src/__tests__/test-helper.ts` is for shared test helpers whereas `/src/server/__tests__test-helper` is for server specific stuff? If so, I like it the thinking... I'm just wondering when `testConfig` would be used by anythign other than server specific stuff?

The React stuff we talked about doesn't need the test config for example

Reply:
Yes, that was the thinking, but that's a great point about the config - we shouldn't ever need that for react stuff. I'll move that over and delete the file if it's empty.